typescript/lib/modules/lambda_context.ts

```diff
@@ -0,0 +1,38 @@
+import { ApiContainerServiceClient } from "../..//kurtosis_core_rpc_api_bindings/api_container_service_grpc_pb";
+import { ExecuteLambdaArgs, ExecuteLambdaResponse } from "../../kurtosis_core_rpc_api_bindings/api_container_service_pb";
+import { newGetExecuteLambdaArgs } from "../constructor_calls";
+import { okAsync, ResultAsync, Result } from "neverthrow";
+import * as grpc from "grpc";
+
+export type LambdaID = string;
+
+// Docs available at https://docs.kurtosistech.com/kurtosis-libs/lib-documentation
+export class LambdaContext {
+	private readonly client: ApiContainerServiceClient;
+	private readonly lambdaId: LambdaID;
+
+    constructor (client: ApiContainerServiceClient, lambdaId: LambdaID) {
+        this.client = client;
+        this.lambdaId = lambdaId;
+    }
+
+    // Docs available at https://docs.kurtosistech.com/kurtosis-libs/lib-documentation
+    public async execute(serializedParams: string): Promise<[string, Error]> {
```

Comment:
Same comment here and everywhere - rather than returning `[string, Error]` we can return `Result<string>`